tensorflow/core/kernels/mkl_fused_ops_test.cc

```diff
@@ -293,10 +295,8 @@ TYPED_TEST_P(MklFusedConv2DWithBiasOpTest, SpatialConvolutionAndRelu) {
   this->VerifyConv2DWithBiasAndRelu(filter_size, filter_count);
 }
 
-REGISTER_TYPED_TEST_CASE_P(MklFusedConv2DWithBiasOpTest,
-                           OneByOneConvolution,         //
-                           SpatialConvolution,          //
-                           OneByOneConvolutionAndRelu,  //
+REGISTER_TYPED_TEST_CASE_P(MklFusedConv2DWithBiasOpTest, OneByOneConvolution,
```

Comment:
Oops. I didn't realize the `//` was to prevent `clang-format` to merge the line. Could you please change back to the original formatting with '//' at the end of the line and one test case per line? I'm so sorry for the inconvenience!